docs/Plugins.md

```diff
@@ -79,37 +79,27 @@ Sometimes, you will need to know when the server is about to close, for example
 
 Do not forget that `register` will always create a new Fastify scope, if you don't need that, read the following section.
 
-If you register an anonymous function using `fastify-plugin`, it will automatically gain a new property based on the file name the `.fp()` method was called from. You can access it using `Symbol.for('fastify.display-name')`. See the example below.
+If you register an anonymous function using `fastify-plugin`, it will automatically set the meta option `name` to the file name the `.fp()` method was called from. You can access it using `fn[Symbol.for('plugin-meta')].name` or `fn[Symbol.for('fastify.display-name')]`. See the example below. If you set the name using the meta property, the file name will be ignored and the `fastify.display-name` will be set to the meta name as well.
```

Comment:
All references to `fastify.display-name` should be removed since it will be removed in the next major version of `fastify-plugin`.

Also, `.fp()` should just be `fp()`.